src/utils.sh

```diff
@@ -462,15 +496,22 @@ function handle_exit() {
 
     if [[ "$LOCAL_E2E" == "1" ]];
     then
-        if [[ $e_code != 0 ]];
-        then
-            echo errors-file -----------------------------
-            cat $STDERR_REDIRECT
-            echo errors-file -----------------------------
-        fi
-        echo stdout-file -----------------------------
-        cat $STDOUT_REDIRECT
-        echo stdout-file -----------------------------
+        iotedge check
+#        if [[ $e_code != 0 ]];
+#        then
+#            echo errors-file -----------------------------
+#            cat $STDERR_REDIRECT
```

Comment:
>cat $STDERR_REDIRECT [](http://example.com/codeflow?start=13&length=20)

if i remember correctly, the goal here was to see the error output in automation as we are not preserving the log file in the cicd pipeline. iotedge check may not provide the same error output.

Reply:
removed